content/v2.0/reference/flux/stdlib/monitor/statechangesonly.md

```diff
@@ -0,0 +1,33 @@
+---
+title: monitor.stateChangesOnly() function
+description: >
+  The `monitor.stateChangesOnly()` function  takes a stream of tables that contains a` _level`
+  column and returns a stream of tables where each record represents a state change.
```

Comment:
change from `fromLevel` to `toLevel` only...or are there other state changes?

Reply:
stateChangesOnly just outputs whenever there's a change. It's not smart enough to know the levels.